projects/addon-mobile/components/sheet-dialog/sheet-dialog.style.less

```diff
@@ -116,7 +116,7 @@
 .t-content {
     position: relative;
     isolation: isolate;
-    padding-block-end: calc(1.5rem + env(safe-area-inset-bottom));
```

Comment:
You need to rebase, pre-commit hook combined two paddings into one.

Also we only need this in fullscreen mode, in regular mode we need to add top padding to the offset of the sheet, not to the padding of its content.